network/src/main/java/org/openjdk/skara/network/RestRequest.java

```diff
@@ -328,6 +328,14 @@ private HttpResponse<String> sendRequest(HttpRequest.Builder request, boolean sk
         while (true) {
             try {
                 response = cache.send(authId, request, skipLimiter);
+                // If the status code is 301(Moved Permanently), follow the redirect link
+                if (response.statusCode() == 301) {
+                    var location = response.headers().firstValue("location");
+                    if (location.isPresent()) {
+                        request = request.uri(URI.create(location.get()));
+                        continue;
```

Comment:
I think we should increment the retryCount here to avoid the possibility of bad server behavior causing us to get stuck in a loop.

Reply:
It's a good point!